src/components/Header.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { Dropdown, Button, Modal } from "flowbite-react";
+import { CadtLogo } from "./CadtLogo";
+import { ExplorerLogo } from "./ExplorerLogo";
+import { TokenizationLogo } from "./TokenizationLogo";
+import { LocaleSwitcher } from "./LocalSwitcher";
+import styled from "styled-components";
+
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+  background-color: ${(props) => props.theme.topBarBgColor};
+`;
+
+const CustomHeader = ({
+  activeApp,
+  appLinks,
+  setActiveApp,
+  showConnect,
+  setShowConnect,
+  handleLocaleChange,
+  handleDisconnect,
+  validateLocalStorage,
+  handleSubmit,
+}) => {
+  const ActiveLogo = activeApp.logo;
+
+  return (
+    <Header
+      className="App-header px-4"
+    >
+      {validateLocalStorage() ? (
```

Comment:
all of this this functionality should remain in App.jsx. the branding logo should be passed to the header via props and the colors should be pulled from the theme and set the header styled component in app.css